cmd/operator-sdk/main.go

```diff
@@ -48,7 +48,7 @@ func main() {
 				"See `operator-sdk init -h` and the following doc on how to scaffold a new project:\n" +
 				"https://v0-19-x.sdk.operatorframework.io/docs/golang/quickstart/\n" +
 				"To migrate existing projects to the new layout see:\n" +
-				"https://sdk.operatorframework.io/docs/golang/migration/project_migration_guide/\n"
+				"https://v0-19-x.sdk.operatorframework.io/docs/golang/project_migration_guide/\n"
```

Comment:
If we want users to upgrade to the latest operator-sdk with the most up-to-date docs, shouldn't this point to the latest guide?
```suggestion
				"https://sdk.operatorframework.io/docs/building-operators/golang/project_migration_guide/\n"
```

Reply:
Sure @estroz . I was going by the update from @camilamacedo86 in the issue https://github.com/operator-framework/operator-sdk/issues/3862. Please confirm if this is the expected update so that I'll make this change.

Reply:
Hi @estroz, 

it is the SDK 0.19.x branch. So, here we should make the tool and all point of to the docs of its version and not master branch. See that the master branch can change.

Reply:
I think @estroz's suggestion makes sense from a content standpoint. Future 1.x updates should all be backward compatible, so the migration from 0.19 should never change again until we get to 2.x.

The one problem that @camilamacedo86 raises is the actual location of the link might change. Perhaps in a follow-up we can look into whether it would be possible to check that link somewhere along the line in our CI process.